zenoh/src/session.rs

```diff
@@ -1044,7 +1091,15 @@ impl Session {
                 res.subscribers.retain(|sub| sub.id != sub_state.id);
             }
 
-            if sub_state.origin != Locality::SessionLocal {
+            #[cfg(not(feature = "unstable"))]
```

Comment:
Locality should be an unstable feature. So both cases should be under the "unstable" feature gate.

Reply:
Locality is public when unstable feature is enabled and private otherwise.